__mocks__/axios.js

```diff
@@ -0,0 +1,9 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2019 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import mockAxios from 'jest-mock-axios';
+export default mockAxios;
```

Comment:
`export default from 'jest-mock-axios';`

Reply:
I don't think you can do that. At least ESLint complains about it.

Reply:
`export {default} from '...';` works for sure. the syntax from my initial comment may still be in a proposal stage and not supported

https://stackoverflow.com/questions/39999282/re-export-default-in-es-6-modules

Reply:
It's an experimental feature.